woocommerce/woocommerce
# Remove manage store activity from home screen note

### All Submissions:






### Changes proposed in this Pull Request:

Removes the note about managing store activity from the home screen.

Part of #34930 






### How to test the changes in this Pull Request:




1. Install a plugin like [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)
1. Under Tools -> Cron Events, run wc_admin_daily
1. Make sure the note with a title of "New! Manage your store activity from the Home screen" has not been added



### Other information:




### FOR PR REVIEWER ONLY:
